src/v/net/connection.cc

```diff
@@ -17,6 +17,7 @@
 #include <seastar/core/future.hh>
 #include <seastar/net/tls.hh>
 
+#include <asm-generic/errno.h>
```

Comment:
i'm pretty sure you should only need `<errno.h>` but the LSP usually doesn't know